app/views/unattended/provisioning_templates/snippet/kickstart_kernel_options.erb

```diff
@@ -67,7 +67,7 @@ description: |
   end
   options.push("dualstack!") if subnet4 && subnet6
   if subnet4 && subnet4.dhcp_boot_mode?
-    options.push("ip=dhcp") if rhel_compatible && major >= 7
+    options.push("ip=::::#{hostname}::dhcp") if rhel_compatible && major >= 7
```

Comment:
Note to myself, from the docs:

> Technically all of the items are optional, so if you want to use dhcp but also set a hostname you can use ip=::::<hostname>::dhcp.

https://anaconda-installer.readthedocs.io/en/latest/boot-options.html